chart/localdb/charts/dbdump/templates/dbdump-job.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: compass-dbdump
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+        release: {{ .Release.Name }}
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+    spec:
+        serviceAccountName: {{ $.Chart.Name }}-dbdump-job
+        restartPolicy: Never
+        shareProcessNamespace: true
+        containers:
+          {{ if eq .Values.global.database.embedded.enabled false }}
+          - name: cloudsql-proxy
+            image: gcr.io/cloudsql-docker/gce-proxy:1.23.0-alpine
+            command:
+              - /bin/sh
+            args:
+              - -c
+              - "trap 'exit 0' SIGINT; echo 'Waiting for istio-proxy to start...' && sleep 15; /cloud_sql_proxy -instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432 -term_timeout=2s"
+          {{ end }}
+          - name: dbdump
+            image: bitnami/postgresql:11.15.0-debian-10-r93
+            imagePullPolicy: IfNotPresent
+            env:
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: compass-postgresql
+                  key: postgresql-director-username
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: compass-postgresql
+                  key: postgresql-director-password
+            - name: DB_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: compass-postgresql
+                  key: postgresql-serviceName
+            - name: DB_PORT
+              valueFrom:
+                secretKeyRef:
+                  name: compass-postgresql
+                  key: postgresql-servicePort
+            - name: DB_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: compass-postgresql
+                  key: postgresql-director-db-name
+            - name: DB_SSL
+              valueFrom:
+                secretKeyRef:
+                  name: compass-postgresql
+                  key: postgresql-sslMode
+            command:
+              - "/bin/bash"
+            args:
+              - "-c"
+              - "sleep 20; [ -f /dbdata/dump.sql ] && PGPASSWORD=${DB_PASSWORD} psql --username=${DB_USER} --host=${DB_HOST} --port=${DB_PORT} --dbname=${DB_NAME} --single-transaction -f /dbdata/dump.sql --set ON_ERROR_STOP=on --set sslmode=${DB_SSL} || echo 'There is no dump.sql to be applied.' ; exit_code=$?; echo '# KILLING PILOT-AGENT #'; pkill -INT cloud_sql_proxy; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
```

Comment:
Why the sleep 20 is needed?

Reply:
It was in migrator, I assume it is because proxy to start, but on second thought - the proxy is not needed, So I will try to remove it (and the sleep).